1.9/usage/service-discovery/marathon-lb/marathon-lb-advanced-tutorial.md

```diff
@@ -241,21 +242,19 @@ To demonstrate the vhost feature:
         ```bash
         Created deployment 5f3e06ff-e077-48ee-afc0-745f167bc105
         ```
-        
-    1.  Deploy the modified nginx external app on DC/OS using this command:
-    
-        ```bash
-        dcos marathon app add nginx-external.json
-        ```
        
     **DC/OS GUI**
     
-    1.  Navigate to the **Services** > **nginx-external** service.
+    1.  Navigate to the **Services > Services > nginx-external** service and click edit.
  
-    1.  Select **Edit** > **Labels** > **Add Label**.
+    1.  Select **Environment > ADD LABEL**.
+    
+    1.  Enter `HAPROXY_0_VHOST` for **KEY** and specify your public DNS name for **VALUE**.
     
         ![Update app](/docs/1.9/usage/service-discovery/marathon-lb/img/nginx-external-gui.png)
         
+        **Important:** Do not include the leading `http://`trailing slash (`/`) in your public DNS name.
```

Comment:
If we were calling it an FQDN, they could Google FQDN and discover this, as well.